README.md

```diff
@@ -172,6 +175,12 @@ jobs:
           MULTILINE_JSON_STRING: ${{ steps.get-installation-repositories.outputs.data }}
 ```
 
+### Run the workflow in a github.com repository against an organization in GitHub Enterprise Server
+
+```yaml
+on: [push]
+```
```

Comment:
This example does not look complete

Reply:
hmm how did that happen 🤣 but yeah let's do a follow up issue